server/index.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../dist'));
+
+const PORT = 3101;
+
+// This version is only used for dev testing
+app.get('/tiersIncluded', (req, res) => {
+  const dataString = { bundleId: 1 };
+  return db.tiersRequest(dataString)
+  .then(tiersData => {
+    if (tiersData) {
+      return tiersData;
```

Comment:
Checking in:
1) You are swallowing your error in your model.
2) If there is an error and it gets console.logged in your model, what gets sent through? Undefined? An empty object? Nothing?
3) Double checking that however/whatever comes through can get passed through your chain and/or will trigger your error conditional.